components/teleport/Teleport.vue

```diff
@@ -163,18 +163,19 @@ const { urlPrefix } = usePrefix()
 const fiatStore = useFiatStore()
 const fromChain = ref(Chain.POLKADOT) //Selected origin parachain
 const toChain = ref(Chain.ASSETHUBPOLKADOT) //Selected destination parachain
-const amount = ref() //Required amount to be transfered is stored here
+const amount = ref(0) //Required amount to be transfered is stored here
 const unsubscribeKusamaBalance = ref()
 const teleportFee = ref()
 
-const nativeAmount = computed(
-  () => (amount.value || 0) * Math.pow(10, currentTokenDecimals.value),
+const nativeAmount = computed(() =>
+  Math.floor(amount.value * Math.pow(10, currentTokenDecimals.value)),
```

Comment:
why floor?

Reply:
was giving decimal value 

![CleanShot 2023-11-28 at 15 36 24@2x](https://github.com/kodadot/nft-gallery/assets/44554284/84ab1694-6a02-4080-8054-64885e0b2567)

Reply:
in this case

![CleanShot 2023-11-29 at 09 59 14](https://github.com/kodadot/nft-gallery/assets/44554284/b54da0e7-327e-4718-9d06-76239044d378) 

![CleanShot 2023-11-29 at 09 54 50@2x](https://github.com/kodadot/nft-gallery/assets/44554284/ad1c37ce-6e25-4848-871d-b482ffcdcf0f)